lib/charms/tls_certificates_interface/v4/tls_certificates.py

```diff
@@ -523,8 +523,8 @@ def to_json(self) -> str:
 
 
 @dataclass(frozen=True)
-class RequirerCSR:
-    """This class represents a certificate signing request requested by the TLS requirer."""
+class RequirerRequest:
```

Comment:
Should we rename this one to `CertificateRequest`?

Reply:
For me having an object called `CertificateRequest` and another one called `CertificateSigningRequest` is confusing.
I think since this object tells us which relation id (requirer) does this certificate request belong to it makes sense to have the term "Requirer" in the name. And to keep it in sync with `ProviderCertificate`
How about `RequirerCertificateRequest`?

Reply:
Yes I'm fine with `RequirerCertificateRequest`, it matches better with the opposite `ProviderCertificate`

Reply:
Renamed